shared/tracker/selector.js

```diff
@@ -9,7 +9,11 @@ export function selector(username: string): (store: Object) => ?Object {
         },
         tracker: {
           trackers: {
-            [username]: store.tracker.trackers[username],
+            [username]: {
+              ...store.tracker.trackers[username],
+              trackers: [],
```

Comment:
this part of the store mutates as we get info on users but this isn't actually consumed by the trackers so lets not pass this down. how this whole system works could be so much simpler and more accurate but likely not worth fixing now. this is pretty simple for now